packages/cli/test/integration/dummy/test1.integration.js

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const {expectToMatchSnapshot} = require('../../snapshots')();
+
+describe('test1', () => {
+  it('matches snapshot 1', () => {
+    expectToMatchSnapshot('test1 result');
+  });
+});
+
+require('./common.integration').commonTest();
```

Comment:
Calling `commonTest` outside of any `describe` block does not make sense to me. It creates duplicate entries in the list of test cases. In my experience with shared test suites, a shared test suite is typically called from inside a `describe` block, to scope each instance of the suite to the particular scenario being tested. 

For example:

https://github.com/strongloop/loopback-next/blob/ae6427322451c914ae54f44dbb656981e7fbbb81/acceptance/repository-cloudant/src/__tests__/cloudant-default-repository.acceptance.ts#L13-L20